pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -229,6 +234,21 @@ sub _centos_symlink_protection {
             }
         );
     }
+    if ($bluehost_ea4) {
+        $security_advisor_obj->add_advice(
+            {
+                'key'        => 'Apache_bluehost_provided_symlink_protection',
+                'type'       => $warn,
+                'text'       => $self->_lh->maketext('Apache Symlink Protection: the Bluehost provided Apache patch is in effect'),
+                'suggestion' => $self->_lh->maketext(
+                    "It appears that the Bluehost provided Apache patch is being used to provide symlink protection. This is less than optimal. Please review [output,url,_1,Symlink Race Condition Protection,_2,_3].",
+                    'https://go.cpanel.net/EA4Symlink',
```

Comment:
@bk2204 It is currently not supposed to point to a valid document, as the documentation's policies and procedures don't permit public exposure of the document until it passes documentation review.  With that in mind, the functionality that turns this on and off (by adding the symlink_protect entry) is not merged upstream, so the risk of exposing the link is minimal.